packit/api.py

```diff
@@ -64,6 +66,23 @@ def get_packit_version() -> str:
         return "NOT_INSTALLED"
 
 
+@dataclass
```

Comment:
Oof, epel8 doesn't seem to like dataclasses, old Python :( Will change it to a regular class with `__init__` and `__eq__`